backend/src/game-stat/dto/create-game-stat.input.ts

```diff
@@ -13,32 +13,32 @@ import {
 @InputType()
 export class CreateGameStatInput {
   @Field(() => EGameType)
-  @IsEnum(EGameType, { message: '$property must be a valid $constraint1.' })
+  @IsEnum(EGameType, { message: '$property must be a valid EGameStat.' })
   type: EGameType
 
-  @Field(() => Int)
-  @IsNotEmpty()
+  @Field(() => Int, { nullable: true })
```

Comment:
Delete nullable true and @IsOptional(), here and for scoreWinner, and scoreLoser as well